recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -107,12 +113,10 @@ EXTRA_OECMAKE += "\
     -DEXTERN_EVENTS="${WPEFRAMEWORK_EXTERN_EVENTS}" \
     -DBUILD_SHARED_LIBS=ON \
     -DRPC=ON \
-    -DVIRTUALINPUT=ON  \
```

Comment:
Could you please tell me why this was removed?

Reply:
It is already available as part of PACKAGECONFIG, hence removed it